openff/toolkit/tests/test_utils.py

```diff
@@ -84,30 +85,6 @@ def test_get_data_file_path():
     assert os.path.exists(filename)
 
 
-@pytest.mark.parametrize(
-    "unit_string,expected_unit",
-    [
-        ("kilocalories_per_mole", unit.kilocalories_per_mole),
-        (
-            "kilocalories_per_mole/angstrom**2",
-            unit.kilocalories_per_mole / unit.angstrom ** 2,
-        ),
-        ("joule/(mole * nanometer**2)", unit.joule / (unit.mole * unit.nanometer ** 2)),
-        ("picosecond**(-1)", unit.picosecond ** (-1)),
-        ("300.0 * kelvin", 300 * unit.kelvin),
-        ("1 * kilojoule + 500 * joule", 1.5 * unit.kilojoule),
-        ("1 / meter", 1.0 / unit.meter),
-    ],
-)
-def test_ast_eval(unit_string, expected_unit):
-    """Test that _ast_eval() correctly parses string quantities."""
-    from openff.toolkit.utils.utils import _ast_eval
-
-    ast_root_node = ast.parse(unit_string, mode="eval").body
-    parsed_units = _ast_eval(ast_root_node)
-    assert parsed_units == expected_unit
```

Comment:
Is this being replaced by something equivalent? The `1 kJ + 500 J` test probably isn't necessary, but the single-value ones like `joule/(mole * nanometer**2)` would be good to keep around.

Reply:
Most of the functionality is tested in the `openff-units` package: https://github.com/openforcefield/openff-units/tree/main/openff/units/tests

I'm not sure what all is and is not transferred over; that tests are passing gives me confidence that most behavior is covered

Reply:
Oh, nice! Glad to see this is already there. Yeah, the `kcal/mole`  vs. `kcal_per_mole` ambiguity is the best thing to test. Really well thought-out tests there :-)